### ISSUE TITLE
 Accessibility Violations - AreaChart and StackedAreaChart

### ISSUE THREAD

**Issue by MichaelMichaelMotorcycle** [carbon-design-system/ibm-security#735]
## Bug - Running the  "IBM Equal Access Accessibility Checker" on a AreaChart or StackedAreaChart  has accessibility violations

**Expected behavior -**

no accessibility violations in base carbon chart features for the AreaChart .

**Actual behavior -**

We are seeing these in our application and can reproduce using the story book example data and configurations.
 
1. On the graph tool tip when it is hidden: 
"Text contrast of 1.00 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400 "

2. On the SVGs in the graph
"The 'role' defined on the element is not valid per WAI-ARIA specification"

roles that are present 
role="graphics-symbol"  and  role="graphics-document document"

### Steps for reproducing

1. Took sandbox code for
  Component:<AreaChart />" https://carbon-design-system.github.io/carbon-charts/react/?path=/story/area--area-time-series
 which is here: https://codesandbox.io/s/yhcf1?file=/src/index.js:0-2142
   moved files as such    ./src/index.js   ./src/ibm-plex-font.css   ./public/index.html   ./package.json
2. yarn install && yarn Dev 
3.  Install and run  "IBM Equal Access Accessibility Checker"  on the page.  I used the chrome extension here.
https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp
4. In the browser web debugger run the accessibility checker 

For the StackedAreaChart I did same steps but I used  the sandbox code for https://carbon-design-system.github.io/carbon-charts/react/?path=/story/area--stacked-area-time-series    which is here https://codesandbox.io/s/1xe7q?file=/src/index.js:0-2085

### Affected browsers

[What's my browser?](http://www.whatsmyua.com) and [browserl.ist supported browsers](http://browserl.ist/?q=%3E+1%25%2C+not+IE+11)

- [ ] Chrome Version 84.0.4147.125  (Official Build) (64-bit)
### Optional information

{
  "dependencies": {
    "@carbon/charts": "^0.35.1",
    "d3": "^5.16.0",
    "react-scripts": "3.0.1",
    "react-dom": "^16.12.0",
    "react": "^16.12.0",
    "carbon-components": "^10.5.0",
    "@carbon/charts-react": "^0.35.1"
  },

**Comment by MichaelMichaelMotorcycle** [comment]
Opened in the wrong repo.
